src/components/Cluster/ClusterDetail/RegionAndVersions.js

```diff
@@ -66,14 +66,7 @@ function RegionAndVersions({
             <>
               <Dot />
               <i className='fa fa-kubernetes' />
-              {(() => {
-                const kubernetes = release.components.find(
-                  (component) => component.name === 'kubernetes'
-                );
-                if (kubernetes) return kubernetes.version;
-
-                return null;
-              })()}
+              {release.kubernetesVersion}
```

Comment:
Could you please undo this change? I'm preparing a PR which rewrites this component in typescript lined up and would like to avoid having merge conflicts

Reply:
If that makes you happy :) Done